### PR TITLE
Adding cookieDomain before Optimizely

### PR DESCRIPTION
**Pull request by adamlofting**
This is WIP - just opening PR so it's easier to talk through.

### CODE REVIEW

File: pages/index.jsx [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+
 var Index = React.createClass({
   render: function() {
     return (
       <html>
         <head>
+          <script>
+            window['optimizely'] = window['optimizely'] || [];
+            window['optimizely'].push(["setCookieDomain", {{cookieDomain}}]);
+          </script>
```
Comment (adamlofting) [inline pages/index.jsx]: This won't work with React, as it gets rendered like this:

```js
<script>window[&#x27;optimizely&#x27;] = window[&#x27;optimizely&#x27;] || []; window[&#x27;optimizely&#x27;].push([&quot;setCookieDomain&quot;, localhost]);</script>
```

What's the best way to add this in React?

@ScottDowne or @alicoding ?

Comment (jbuck) [inline pages/index.jsx]: Out of curiosity, why do you need to set the subdomain? Isn't it automatically scoped to the correct domain?

Comment (adamlofting) [inline pages/index.jsx]: It works by default, but we're being extra careful:
https://bugzilla.mozilla.org/show_bug.cgi?id=1125995

Comment (jbuck) [inline pages/index.jsx]: Ahhh, I see. So optimizely sets the cookie domain to mozilla.org by default. laaaaaaame

Comment (alicoding) [inline pages/index.jsx]: Can you also move it to `componentDidMount` instead?

File: pages/index.jsx [hunk 2]
```diff
@@ -1,10 +1,20 @@
 var React = require('react');
 
+var Habitat = require('habitat');
+Habitat.load();
```
Comment (ScottDowne) [inline pages/index.jsx]: Woah, this works client side?

Comment (adamlofting) [inline pages/index.jsx]: I assumed this index.jsx file was just being run in the pre-compile. And we'd end up with static HTML... but I might have missed something important here.

Comment (alicoding) [inline pages/index.jsx]: I would say use this way instead? https://github.com/mozilla/id.webmaker.org/blob/develop/webpack.config.js#L42-L50

**Comment by ScottDowne** [comment]
@adamlofting this still WIP?

**Comment by adamlofting** [comment]
> @adamlofting this still WIP?

Yep, I started on some fixes to this but didn't get time to finish it today.

**Comment by adamlofting** [comment]
I ended up taking a different approach, so will open a new PR.

**Pull request closed without merging**